docs/features/features-walk-through/restricted_records.md

```diff
@@ -0,0 +1,22 @@
+# Restricted records
+
+A record can be marked as restricted in order to grant access to it only to a specific group of users or organisations. This is useful for example if you want to share a record with a colleague or team before making it public.
```

Comment:
Using the actual naming we have in the requests, i.e. `users` and `groups` (a group can be an organization, but also many others, for example when using ldap/egroups).

```suggestion
A record can be marked as restricted in order to grant access to it only to a specific users or groups. This is useful for example if you want to share a record with a colleague or team before making it public.
```